Katas/BankAccount/README.md

```diff
@@ -36,44 +42,64 @@ Run every test with the associated shortcut and check the result.
 
 ## Scenarios
 
-There are different scenarios that will form the overall behavior of the bank account.
+There are different scenarios that will form the overall behavior of a bank account.
 
 ### Scenario 1
 
+> Our Bank needs accounts, but every new account will start without any deposit.
+
 Given a new bank account, when the bank account is newly created, then the initial amount is zero.
 
 ### Scenario 2
 
-Given a bank account, when deposing a given amount to an account, then this amount is added to accounts balance.
+> This customer wanted to deposit his money, but our system did not let him - even when he has an account at our bank.
```

Comment:
It might be possible, that this could be misunderstood. I might sound like a requirement. "Create a bank account, the customer is not able to deposit to it"

Reply:
It should sound like a requirement, so the programmer/dojo participants can convert a spoken requirement to a useful requirement by changing its structure.

Reply:
Ok, let's give it a try.